shared/profile/prove-enter-username.desktop.js

```diff
@@ -81,9 +81,6 @@ class Render extends Component<void, Props, State> {
   }
 
   handleUsernameChange (username: string) {
-    if (this.props.onUsernameChange) {
```

Comment:
Yes, it does break BTC. I'll update with a fix for that.

It was removed from triggering every time the user typed to only when the user clicked submit. Reason being so that the hostname extraction stuff wasn't running every time and storing the extracted hostname state in a different part of the store. I could add a separate step to the submitUsername action that does some hostname cleanup prior to sending it to the service/storing it for the next screen, or we could move the BTC validation and other validation stuff into the components internal state.